src/OpenTelemetry/Metrics/MetricStreamConfiguration.cs

```diff
@@ -38,6 +38,18 @@ public class MetricStreamConfiguration
 
         public string[] TagKeys { get; set; }
 
+        /// <summary>
+        /// Gets or sets the names of <see cref="Baggage"/> keys, which,
+        /// if available, will be used as additional tags for Metric
+        /// aggregation.
+        /// <see cref="Baggage.GetBaggage(string)"/> will be used by the SDK
+        /// internally to obtain the additional tags.
+        /// </summary>
+        /// <remarks>
+        /// It is invalid to specify this for Observable instruments.
+        /// </remarks>
+        public string[] BaggageKeysForAdditionalTags { get; set; }
```

Comment:
I'm not sure if these should be called Tags or Dimensions?

It seems the word "tag" could mean two different things:
1. The .NET way of calling an Attribute.
2. The Prometheus way of calling a "dimension".

Reply:
Given Baggage only take string not Attribute, I guess "tag" will introduce confusion.